### ISSUE TITLE
Reveal just one square...

### ISSUE THREAD

**Issue by erikstar** [viresh-ratnakar/exolve#39]
A feature request... I just think I myself have situations where I want to get as little help as possible...

/Erik S again

**Comment by viresh-ratnakar** [comment]
Yeah, this one I will add. I will add a toggle in the buttons panel to make "this" mean just the current cell as opposed to all the active cells, and it would affect "Clear this" "Check this" and "Reveal this"

**Comment by erikstar** [comment]
Super!

**Comment by viresh-ratnakar** [comment]
I've added this (in v0.90), but there is a caveat.

I tried adding a new toggle near the buttons, but it looked awkward (especially as it's hard to express its functionality concisely). I have instead made it so that a *long* click on "Check this" or "Reveal this" turns them into "Check cell" and "Reveal cell" for that particular click. This seems to work nicely and feels intuitive to me. I've also added hover-text to these buttons so that users may discover this feature.

The caveat is that on phones and tables (I only tested on Android ones), long-click does not work (Android turns the long touch event into a longpress event and never even fires the mouse events that I need). I'll try to fix that at some point by trying to look for touch events properly.
